components/org.wso2.carbon.identity.oauth/src/main/java/org/wso2/carbon/identity/oauth2/OAuth2Service.java

```diff
@@ -241,7 +241,7 @@ public OAuth2ClientValidationResponseDTO validateClientInfo(String clientId, Str
                 validationResponseDTO.setPkceSupportPlain(appDO.isPkceSupportPlain());
                 return validationResponseDTO;
             } else {    // Provided callback URL does not match the registered callback url.
-                log.warn("Provided Callback URL does not match with the provided one.");
+                log.warn("Provided Callback URL does not match with the registered one.");
```

Comment:
`Provided Callback URL does not match with the registered URL` WDYT?

Reply:
If this is good shall we fix other places as well?

Reply:
Less confusing. Will fix other places as well.

Reply:
https://github.com/wso2-extensions/identity-inbound-auth-oauth/pull/2031